src/PProf.jl

```diff
@@ -64,11 +64,16 @@ the web-server to use the new output.
 If you manually edit the output file, `PProf.refresh()` will refresh the server without
 overwriting the output file. `PProf.kill()` will kill the server.
 
+You can also use `PProf.refresh(file="...")` to open a new file in the server.
+
 # Arguments:
-- `data::Vector{UInt}`: The data provided by `Profile.fetch` [optional].
-- `period::UInt64`: The sampling period in nanoseconds [optional].
+- `data::Vector{UInt}`: The data provided by `Profile.retrieve()` [optional].
+- `lidict::Dict`: The lookup dictionary provided by `Profile.retrieve()` [optional].
+    - Note that you need both the `data` and the `lidict` returned from
+      `Profile.retrieve()` in order to export profiling data between julia processes.
 
 # Keyword Arguments
+- `sampling_delay::UInt64`: The period between samples in nanoseconds [optional].
```

Comment:
Seems to be missing from the docstring signature now? :)

Reply:
Thanks! :) Fixed in latest commit